soil_params_Gupta2020_2022/create_artifacts.jl

```diff
@@ -0,0 +1,95 @@
+# Ksat data derived by
+# Gupta, S., Lehmann, P., Bonetti, S., Papritz, A., and Or, D., (2020)
+# Global prediction of soil saturated hydraulic conductivity using random
+# forest in a Covariate-based Geo Transfer Functions (CoGTF) framework.
+# Journal of Advances in Modeling Earth Systems, 13(4), e2020MS002242.
+# https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2020MS002242
+
+# Water retention curve parameters derived by
+# Gupta, S., Papritz, A., Lehmann, P., Hengl, T., Bonetti, S., & Or, D. (2022). 
+# Global Mapping of Soil Water Characteristics Parameters—Fusing Curated Data with 
+# Machine Learning and Environmental Covariates. Remote Sensing, 14(8), 1947.
+
+# First, navigate to https://zenodo.org/records/3935359 and https://zenodo.org/records/6348799 
+# and download saturated hydraulic conductivity data (former) and water retention
+# parameter data (latter)
+# stored in the .tif files there. There are four files per variable, corresponding to 
+# four different soil depths.
+
+# The following `filedir` is hard-coded to the Caltech cluster, but can be replaced with paths to the tif files
+# on your local machine.
+
+using TiffImages
+using NCDatasets
+using Statistics
+using ClimaArtifactsHelper
+
+filedir = "/groups/esm/ClimaArtifacts/artifacts/soil_params_Gupta2020_2022/raw_data"
```

Comment:
This assumes that the file is a specific location. I'd suggest we leave this script incomple, or set this to `Missing` so that it is clear that one has to fetch the file independently.